packages/pocket-ic/src/management_canister.rs

```diff
@@ -28,6 +28,7 @@ pub struct DefiniteCanisterSettings {
     pub wasm_memory_limit: candid::Nat,
     pub memory_allocation: candid::Nat,
     pub compute_allocation: candid::Nat,
+    pub wasm_memory_threshold: candid::Nat,
```

Comment:
this file should stay in sync with the interface spec so this shouldn't be added before publicly releasing the feature

Reply:
Thanks, done.